SqlServerLib/SQLServer.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <iostream>
+#include "SQLConnection.h"
+#include "../CPPLogger/CPPLogger.h"
+#include "../Crypto/SymmetricCrypto.h"
+
+class SQLServer : public SQLConnection
+{
+public:
+
+	SQLServer() = default;
+	SQLServer(const ConnectParams& connect_string);
+	~SQLServer();
+
+	bool Connect(const ConnectParams& connect_string) override;
+	void InsertEmail(const Email& email) override;
+	void InsertMessage(const Message& message, const Email& email) override;
+	void SelectUsers() override;
+	void ClearTable(const std::string& table) override;
+	bool Disconnect() override;
+
+protected:
```

Comment:
Why we need this members as protected? If we not inherit from this class, we should make them private.